src/fusion-rates-gamow.md

```diff
@@ -0,0 +1,861 @@
+
+
+```python
+Ues = np.linspace(0, 200, 100)
+gamow_many_Ue = []
+
+for Ue in Ues:
+    V_eff = V_nuc(r_fm, state_5S["V0"], state_5S["r_S"], state_5S["a_S"]) + V_cent(r_fm, state_5S["L"]) + V_screened_mol(r_fm, Ue)
+    G = calculate_G(r_fm, V_eff, 74000)
+    exp2G = np.exp(-2*G)
+    Gamma = state_5S["$v_{nuc}/v_{mol}$"] * exp2G * state_5S["gamma"]
+    gamow_many_Ue.append({
+        "$U_e$": Ue,
+        "$G$": G,
+        "$e^{-2G}$": exp2G,
+        "$\Gamma$": Gamma
+    })
+
+# Convert the list of dictionaries to a DataFrame
+gamow_many_Ue = pd.DataFrame(gamow_many_Ue)
+
+# Display the first few rows of the DataFrame
+gamow_many_Ue.head()
+```
+
+Now, normalising to the Koonin value
+
+```python
+Koonin_norm = 3e-64/gamow_many_Ue[gamow_many_Ue["$U_e$"]==0]["$\Gamma$"].values[0]
+gamow_many_Ue["$\Gamma$ (normalised)"] = gamow_many_Ue["$\Gamma$"]*Koonin_norm
+```
+
+```python
+plt.plot(gamow_many_Ue["$U_e$"], gamow_many_Ue["$\Gamma$ (normalised)"])
+
+# Closest values to 150
+target_values = [150]
+closest_points = gamow_many_Ue.iloc[[abs(gamow_many_Ue["$U_e$"] - target).idxmin() for target in target_values]]
+
+# Adding vertical and horizontal lines for the closest points
+for _, row in closest_points.iterrows():
+    U_value = row["$U_e$"]
+    gamma_value = row["$\Gamma$ (normalised)"]
+    # Vertical line from bottom to the curve
+    plt.plot([U_value, U_value], [1e-170, gamma_value], color='gray', linestyle='--')
+    # Horizontal line from the vertical line to the y-axis
+    plt.plot([0, U_value], [gamma_value, gamma_value], color='gray', linestyle='--')
+
+plt.yscale('log')  # Set the y-axis to logarithmic scale
+plt.yticks([1e-20, 1e-30, 1e-40, 1e-50, 1e-60, 1e-70, 1e-80])
+
+plt.xticks([0, 25, 50, 75, 100, 125, 150])
+
+plt.xlabel('Screening energy (eV)')
+plt.ylabel('Fusion rate (s$^{-\mathregular{1}}$)')
+
+plt.xlim([0, 160])
+plt.ylim(ymin=1e-70)
+
+plt.tight_layout()
+# plt.savefig("fusion-rate-screening.png", dpi=600)
+plt.show()
+
+# # COMMENTED OUT CODE CAN BE USED TO REPRODUCE THE FORMAT OF PLOTS FROM https://doi.org/10.1088/1367-2630/ad091c
+# fig, ax = plt.subplots(figsize=(8, 6))
+# plt.yticks([1e-20, 1e-30, 1e-40, 1e-50, 1e-60, 1e-70, 1e-80], [f"{tick:.0E}" for tick in y_ticks])
+# plt.xlabel('Screening energy (eV)', fontsize=25, fontname='Arial')
+# plt.ylabel('Fusion rate (s$^{-\mathregular{1}}$)', fontsize=25, fontname='Arial')
+# plt.xticks(fontsize=25, fontname='Arial')
+# plt.yticks(fontsize=25, fontname='Arial')
+# ax = plt.gca()
+# ax.spines['top'].set_visible(False)
+# ax.spines['right'].set_visible(False)
+```
+
+For a metal like palladium, $U_e \approx 150 \, \rm eV$. This suggests that fusion rates for deuterium inside Palladium could be about 30 orders of magnitude larger than for $D_2$ in a gas.
```

Comment:
For a metal like palladium, $U_e has been estimated as $U_e \approx 150 \ (Huke et al. 2008).